packages/venia-ui/lib/components/Image/image.css

```diff
@@ -17,15 +17,15 @@
 }
 
 .loaded {
-    display: block;
     position: absolute;
     top: 0;
     left: 0;
+    visibility: visible;
 }
 
 .notLoaded {
     composes: loaded;
-    display: none;
+    visibility: hidden;
```

Comment:
Approving this PR, but for posterity, I have to note here that the way this was done (originally) was not the recommended approach.

- the styles on `.loaded` should be on `.image`
- `.loaded` should compose `.image`
- `.notLoaded` should not compose `.loaded`
- `. notLoaded` should compose `.image` and override `visibility`
- (most importantly) `.image` should not be applied directly to the component